catalog/app/components/Markdown/Markdown.js

```diff
@@ -136,7 +140,11 @@ export const getRenderer = memoize(({
     disable: !images,
     process: processImg,
   }));
-  return md;
+  const purify = createDOMPurify(window);
+  // sanitize the HTML
+  this.render =  (data) => purify.sanitize(md.render(data), SANITIZE_OPTS)
```

Comment:
this is not a constructor, so `this` points to the `global` object here, which is not what you want, i guess